.eslintrc

```diff
@@ -6,7 +6,8 @@
         "jest": true
     },
     "rules": {
-        "@typescript-eslint/no-explicit-any": ["off"]
+        "@typescript-eslint/no-explicit-any": ["off"],
+        "new-cap": ["off"]
```

Comment:
Removed the rule for the ability to write [conditions](https://github.com/gravity-ui/onboarding/pull/56/commits/45f94096860e729e91a0335822657e0574d0f939#diff-8772b055e51f3e1b014a8e406b20c082a647e238506e50cc16240a6d049bf569). For example, ShowOncePerMonths